README.md

```diff
@@ -176,6 +177,40 @@ Jump to competencies for a particular level:
 
 ## FAQ
 
+### Why these competencies?
+
+We've found that these four competency areas capture much of what's worked well for us.
+The attributes espoused here have been what we've commonly seen help engineers thrive at Codecademy and similar engineering organizations.
+
+We believe a team should prosper and grow _as a team_: that its success is measured by both the quality and throughput of its output _and_ the personal growth and enjoyment of its individual members.
+Strong technical achievements are only half of that.
+Fostering a collaborative community is also necessary for us to be a great place to work.
+
+#### Why Software Engineering & Design?
+
+To be a strong engineer, you need strong development chops.
+That spans the gamut of the core tasks you'll be doing here, from the granular act of writing code to the larger-scale process of designing and maintaining systems.
+We actively appreciate that our industry is rapidly evolving and new best practices and standards are always around the corner.
+We need engineers who bring in those fresh ideas to be able to produce our best work.
+
+#### Why Execution & Results?
+
+On top of the core development fundamentals, successful engineers are ones who ship products and are able to reliably do so on time with high quality.
+They need to balance the various tasks they're given and, to an extent, manage their own time.
+Engineers have uniquely deep insight into their work, and often need to drive it forward.
+
+#### Why Collaboration & Communication?
+
+Teams are more than the sum of their parts, and as an engineer, part of your responsibilities are that everyone is given the resources to work their best.
+On one level, we need team members to be active participants in bettering the code and documentation around them.
```

Comment:
These next two points ("on one level...", "on a higher level...") seem more like a plug for community + citizenship; I'd suggest rewording to something along the lines of:

"Strong team members both communicate the current progress and blockers of their own work, in order to keep their peers from getting stuck, and share feedback regularly that shape the code and product to be the best it can be."